web/src/views/SubmissionPortal/HarmonizerView.vue

```diff
@@ -544,6 +547,11 @@ export default defineComponent({
             mdi-refresh
           </v-icon>
         </v-btn>
+        <v-snackbar v-model="snackbar"
```

Comment:
I'm seeing some console errors due to the fact that `snackbar` is a `ComputedRef` and is therefor read-only (unless we write a [setter](https://vuejs.org/guide/essentials/computed.html#writable-computed) for it.

Generally, I'm not a fan of writeable computed properties, since it kind of breaks the whole idea of them being derived from other state.

I think the best course of action would be to use a non-computed ref (i.e. `const snackbar = ref(false);`) and update that to `true` in the `validate` function if ALL tabs are valid. That function would also have to set `snackbar` to `false` in the event that it finds validation errors.